### PR TITLE
Tweak scripts to support multiple token owners and transfers

### PR DESCRIPTION
**Pull request by dave**

### CODE REVIEW

**Comment by dave** [comment]
Hey @pheuberger here's some changes that we need to support the full end-to-end tests (with reverse bridge direction)... I don't think these will break anything in this repo!

**Comment by dave** [comment]
It will probably need some rebasing into your existing PR, but I thought I'd put it here so we know what's needed.

**Comment by dave** [comment]
Re: https://github.com/rnd-inc/toucan-bridge-service/pull/130

**Comment by pheuberger** [comment]
should I review this already or do you want to rebase first? :)

**Comment by dave** [comment]
I'll rebase today.

**Comment by dave** [comment]
OK @pheuberger that's rebased now.

File: lib/toucan.js [hunk 1]
```diff
@@ -159,24 +159,43 @@ async function deployTco2(contracts, vintageId) {
 }
 
 async function mintBatch(cBatches, batch, vintageId, admin, broker) {
+	let batchBroker = broker;
```
Comment (pheuberger) [inline lib/toucan.js:162]: you don't need to go this route. function parameters are not `const`, so you can directly assign to `broker`

File: scripts/deploy-local.js [hunk 2]
```diff
@@ -4,21 +4,21 @@ const { deployBridge } = require("../lib/bridge");
 const { prepareToucanEnv } = require("../lib/toucan");
 
 async function deploy() {
-	const [owner] = await hre.ethers.getSigners();
-	const regenBridgeAccount = "0x11241e35B3f79099123aA0C1C4c97b1FcdCd21f6";
```
Comment (pheuberger) [inline scripts/deploy-local.js:8]: why was this actually hardcoded before?

File: lib/toucan.js (hunk 1 above)
Comment (dave) [reply lib/toucan.js:162]: but what if some batches wanted to use the default broker, and some had an override?

File: scripts/deploy-local.js (hunk 2 above)
Comment (dave) [reply scripts/deploy-local.js:8]: Not 100% sure why I hardcoded it... I don't think I know much about how it was working so I think I assumed that account was somehow needed... but definitely wasn't needed to be that particular account.

File: lib/toucan.js (hunk 1 above)
Comment (pheuberger) [reply lib/toucan.js:162]: what I mean is you don't need to use the `batchBroker` variable as you can simply assign to `broker` if you want to override

File: scripts/deploy-local.js (hunk 2 above)
Comment (pheuberger) [reply scripts/deploy-local.js:8]: cool, got it!

**Comment by pheuberger** [comment]
can you fixup your review tweaks into the commits you're fixing? :)

**Comment by dave** [comment]
Done!

**Review by pheuberger — APPROVED** [review]
awesome, thank you

**Pull request merged**